docs/directory-structure.md

```diff
@@ -0,0 +1,56 @@
+# CSC Directory Structure
+
+The CSC repository contains three types of content: Service Accellerator Templates, Infrastructure as Code, and Automated Testing.
+
+This document is designed to establish a standard layout that each content type should meet when being contributed to the repository.
+
+## Service Accellerator Templates
+
+**Parent directory:**
+
+1. Cloud service provider name
+1. `service_accellerator_templates/`
+1. Name of topic/category
+
+*Example:*
+`aws/service_accellerator_templates/dynamodb`
+
+**Accepted filetypes:** `.docx`, `.md`
+
+## Infrastructure as Code
+
+**Parent directory:**
+
+1. Cloud service provider name.
+1. Name of IaC format (currently only Terraform is being used)
+
+**Terraform directory organization:**
+
+Terraform contributions should always take advantage of [modules](https://www.terraform.io/docs/language/modules/develop/index.html). Modules should live in `<CSP>/terraform/modules`. Containing all modules for a cloud provider together will help prevent unecessary code duplication.
+
+One or more modules will be used within a configuration, which should live within it's own directory that is named based on the topic.
+
+*Example:*
+
+```bash
+gcp/
+    service_accellerator_templates/
```

Comment:
Typo : `service_accellerator_templates/` should be `service_accelerator_templates/`